lib/helpers/backend/googleBooksSearch.ts

```diff
@@ -0,0 +1,20 @@
+// Note that I was having trouble with https://www.npmjs.com/package/google-books-search
```

Comment:
There is also https://www.npmjs.com/package/@googleapis/books, which appears to be official (from Google), but if the code you have works fine, then I am okay using it.

Reply:
Thanks for letting me know! Used in latest commit :)